implementations/c/config/vault/ockam_vault_hw_cfg.h

```diff
@@ -0,0 +1,69 @@
+ * @brief   
+ ********************************************************************************************************
+ */
+
+#ifndef OCKAM_VAULT_HW_CFG_H_
+#define OCKAM_VAULT_HW_CFG_H_
+
+/*
+ ********************************************************************************************************
+ *                                             INCLUDE FILES                                            *
+ ********************************************************************************************************
+ */
+
+#include <common/include/ockam_def.h>
+
+
+/*
+ ********************************************************************************************************
+ *                                                DEFINES                                               *
+ ********************************************************************************************************
+ */
+
+#define VAULT_MICROCHIP_IO_KEY_ENABLED              OCKAM_FALSE
+
+
+/*
+ ********************************************************************************************************
+ *                                               CONSTANTS                                              *
+ ********************************************************************************************************
+ */
+
+/*
+ ********************************************************************************************************
+ *                                               DATA TYPES                                             *
+ ********************************************************************************************************
+ */
+
+/*
+ ********************************************************************************************************
+ *                                          FUNCTION PROTOTYPES                                         *
+ ********************************************************************************************************
+ */
+
+/*
+ ********************************************************************************************************
+ *                                            GLOBAL VARIABLES                                          *
+ ********************************************************************************************************
+ */
+
+/*
+ ********************************************************************************************************
+ *                                           GLOBAL FUNCTIONS                                           *
+ ********************************************************************************************************
+ */
+
+/*
+ ********************************************************************************************************
+ *                                            LOCAL FUNCTIONS                                           *
+ ********************************************************************************************************
+ */
```

Comment:
remove these place holder comments from all files, we can add them wherever we really need them

Reply:
I'd disagree. Having the comment blocks here ensure code is written in a consistent fashion. In ever file we know where the global variables, data types, etc. are located.